README.md

```diff
@@ -139,6 +139,77 @@ tox -e kubeflow-remote
 tox -e kubeflow-local
 ```
 
+### Run behind proxy
```

Comment:
Before running the uats for kserve, we must set the proxy configs in [kserve-controller](https://github.com/canonical/kserve-operators/blob/main/charms/kserve-controller/config.yaml#L38-L49) and [knative-serving](https://github.com/canonical/knative-operators/blob/main/charms/knative-serving/config.yaml#L56-L67) charms.

My suggestions are either:
1. Add it a prerequisite in the README.md before running using notebook and driver
2. In the notebook, add a cell with pylib juju to set the charm configs only if the proxy envs are set in the notebook pod
wdyt?

Reply:
done in #112